pkg/runtime/application_config_provider.go

```diff
@@ -56,7 +58,11 @@ func (p *ApplicationConfigurationProvider) GetDbConfig() interfaces.DbConfig {
 }
 
 func (p *ApplicationConfigurationProvider) GetTopLevelConfig() *interfaces.ApplicationConfig {
-	return flyteAdminConfig.GetConfig().(*interfaces.ApplicationConfig)
+	applicationConfig := flyteAdminConfig.GetConfig().(*interfaces.ApplicationConfig)
+	if applicationConfig.AsyncEventsBufferSize == 0 {
```

Comment:
no need to do this, just set the default in the config like so
https://github.com/flyteorg/flytepropeller/blob/master/pkg/controller/config/config.go#L47

Reply:
@kumare3 done, ty!